olp-cpp-sdk-dataservice-read/src/generated/api/MetadataApi.h

```diff
@@ -51,6 +51,8 @@ class MetadataApi {
   using CatalogVersionResponse =
       client::ApiResponse<model::VersionResponse, client::ApiError>;
 
+  using CatalogVersionCallback = std::function<void(CatalogVersionResponse)>;
+
```

Comment:
There are the same definitions inside external/olp-edge-sdk-cpp/upstream/olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/Types.h
Why do you need to define here your own same aliases?